cjs/index.js

```diff
@@ -73,12 +72,12 @@ module.exports = ({types: t}, options) => {
             name === (pragma || 'React.createElement')
           ) {
             const {callee} = path.parentPath.node;
-            if (callee && getCalleeName(callee) === interpolation())
+            if (callee && getCalleeName(callee) === interpolationName())
               return;
             path.parentPath.replaceWith(
               t.inherits(
                 t.callExpression(
-                  toMemberExpression(interpolation()),
+                  t.memberExpression(...interpolation()),
```

Comment:
This doesn't work if the expression has more than two pieces (e.g. `a.b.c`); you need to use reduce.

Reply:
fixed ... also fixed spread which wasn't really working